### PR TITLE
Identity Protection/VPN: grammar, links & spacing

### LINKED ISSUE #6556: Traffic filter sentence incomplete
On this page https://docs.microsoft.com/en-us/windows/security/identity-protection/vpn/vpn-security-features#traffic-filters

This sentence is incomplete:

"Network admins to effectively add interface specific firewall rules on the VPN Interface"

It should probably read:

"Traffic filters allow network admins to effectively add interface specific firewall rules on the VPN Interface"

### PR DESCRIPTION
**Pull request by illfated**
**Description:**

As reported in issue ticket #6556 (**Traffic filter sentence incomplete**), there is a missing part in the sentence "Network admins to effectively add interface specific firewall rules on the VPN Interface." to make it work as a full descriptive sentence in this context. This PR aims to correct this issue, in addition to various other adjustments.

Thanks to @klishb for reporting this issue.

**Changes proposed:**
- Add the missing part of the Traffic Filters sentence
- Update 2 outdated and permanently redirected MSDN links
- Replaced "such that" with "to allow", rebuilding the sentence structure
- Uppercase adjustments for "Traffic filters" & "Lockdown"
- Added 3 readability commas, to gain more readability focus
- Add MarkDown indent marker compatibility spacing in the Note blob
- Reduce bullet point spacing from 3 to 1 in the "Applies to" section
- Remove all redundant end-of-line spacing, except for 1 in metadata
- Add missing space after the corrected sentence (after the period)

**Ticket closure or reference:**

Closes #6556

### CODE REVIEW

**Comment by illfated** [comment]
@JohanFreelancer9 : May I ask you to perform a copy review on this PR? Feel free to review as much of the document as you like, because I have noticed a few sentences being a bit awkward in its grammar or sentence structure, but I could not decide exactly how to make the necessary improvements for them.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Just so I don't forget to mention this as well: the ticket author, @klishb, offered the following phrasing for the sentence in question:
> Traffic filters allow network admins to effectively add interface specific firewall rules on the VPN Interface.

Feel free to suggest this sentence form instead, if you find this phrasing to convey its message more correctly.

File: windows/security/identity-protection/vpn/vpn-security-features.md [hunk 1]
```diff
@@ -29,51 +29,50 @@ A VPN profile configured with LockDown secures the device to only allow network
...
 
-There can be many sets of rules which are linked by OR. Within each set, there can be app-based rules and traffic-based rules; all the properties within the set will be linked by AND. In addition, these rules can be applied at a per-app level or a per-device level. 
+There can be many sets of rules which are linked by OR. Within each set, there can be app-based rules and traffic-based rules; all the properties within the set will be linked by AND. In addition, these rules can be applied at a per-app level or a per-device level.
 
-For example, an admin could define rules that specify: 
+For example, an admin could define rules that specify:
 
-- The Contoso HR App must be allowed to go through the VPN and only access port 4545. 
+- The Contoso HR App must be allowed to go through the VPN and only access port 4545.
 - The Contoso finance apps is allowed to go over the VPN and only access the Remote IP ranges of 10.10.0.40 - 10.10.0.201 on port 5889.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/vpn/vpn-security-features.md:70]:
```suggestion
- The Contoso finance apps are allowed to go over the VPN and only access the Remote IP ranges of 10.10.0.40 - 10.10.0.201 on port 5889.
```

**Comment by illfated** [comment]
@JohanFreelancer9 : Thank you for reviewing. Did you compare the sentence suggested by the ticket author with the one I put into the source text? See also copy below:
> Traffic filters allow network admins to effectively add interface specific firewall rules on the VPN Interface.

> Network admins can use Traffic Filters to effectively add interface specific firewall rules on the VPN Interface.

**Comment by illfated** [comment]
By the way, if you find that my latest commit makes to much of a change, or changes the sentence focus too much, feel free to comment and tell me about it.

**Comment by JohanFreelancer9** [comment]
@illfated I much prefer the sentence you put into the source text, although either would have been fine. As far as your last commit goes, I have no problem with it. Looks good.

**Comment by illfated** [comment]
👍 Thank you very much. Having someone confirming what works and what is less so, is invaluable to keep from living in one's own mind world. 😉

**Pull request merged**